umbraco/Umbraco.CMS.Backoffice
# Tiptap extension: Table

## Description

Adds and enables the Table feature for Tiptap editor.
<https://tiptap.dev/docs/editor/extensions/nodes/table>

This PR adds a basic toolbar button and CSS styles for the table layout.

## Types of changes


## How to test?

Check that the Table toolbar button appears, pressing the button will insert a default table with 3 columns, 3 rows (including a table header).

Try copy-pasting a HTML table in from another source, even from here, try this table...

| Version    | Release date      |
| ---------- | ----------------- |
| Umbraco 15 | November 14, 2024 |
| Umbraco 16 | June 12, 2025     |
| Umbraco 17 | November 27, 2025 |
